res/controllers/Hercules-DJ-Console-RMX-scripts.js

```diff
@@ -450,6 +450,19 @@ HerculesRMX.Deck.prototype.cueSelectHandler = function(value) {
    }
 };
 
+HerculesRMX.Deck.prototype.sourceSelectHandler = function(value) {
```

Comment:
If you can point me to an example button that makes use of the approach you're suggesting, I could try to test it later.